packages/react-server-dom-webpack/npm/client.node.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
+var n, w;
 if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./cjs/react-server-dom-webpack-client.node.production.js');
+  n = require('./cjs/react-server-dom-webpack-client.node.production.js');
+  w = require('./cjs/react-server-dom-webpack-client.node-webstreams.production.js');
 } else {
-  module.exports = require('./cjs/react-server-dom-webpack-client.node.development.js');
+  n = require('./cjs/react-server-dom-webpack-client.node.development.js');
+  w = require('./cjs/react-server-dom-webpack-client.node-webstreams.development.js');
 }
+
+exports.registerServerReference = function (r, i, e) {
+  return w.registerServerReference(n.registerServerReference(r, i, e), i, e);
+};
+exports.createServerReference = function (i, c, e, d, f) {
+  return w.registerServerReference(
+    n.createServerReference(i, c, e, d, f),
```

Comment:
I missed this in #33442, but `c, e, d, f` are all ignored for the edge and node builds, aren't they? The original function with all params only seems be exported in the browser build, if I'm not mistaken.